pages/common/tslint.md

```diff
@@ -9,7 +9,7 @@
 
 - Lint on a given set of files:
 
-`tslint {{filename}}.js {{filename1}}.js`
+`tslint {{path/to/file}}.js {{path/to/file1}}.js`
```

Comment:
```suggestion
`tslint {{path/to/file1.js path/to/file2.js ...}}`
```

Reply:
Fixed via script. If seems that it's the only one mistake with the first missing number and the second `1` now. Script did not detected other such errors. 👍🏻